src/Symfony/Component/Translation/README.md

```diff
@@ -3,10 +3,28 @@ Translation Component
 
 The Translation component provides tools to internationalize your application.
 
+Getting Started
+---------------
+
+```
+$ composer require symfony/translation
+```
+
+```php
+use Symfony\Component\Translation\Translator;
+
+$translator = new Translator('fr_FR');
+$translator->addResource('array', [
+    'Hello World!' => 'Bonjour!',
+], 'fr_FR');
+
+echo $translator->trans('Hello World!');
```

Comment:
```suggestion
echo $translator->trans('Hello World!'); // outputs « Bonjour ! »
```

Reply:
Thanks,  I didn't know French punctuation was this different from English/Dutch. Fixed it :)